src/RNTuple/displays.jl

```diff
@@ -11,26 +11,21 @@ function Base.show(io::IO, f::AliasRecord)
     print(io, "AliasRecord(physical_id=$(f.physical_id), field_id=$(f.field_id))")
 end
 
-function Base.show(io::IO, f::ClusterSummary)
-    print(io, "ClusterSummary(num_first_entry=$(f.num_first_entry), ")
-    print(io, "num_entries=$(f.num_entries))")
-end
-
-function Base.show(io::IO, f::FieldRecord)
-    print(io, "parent=$(lpad(Int(f.parent_field_id), 2, "0")), ")
-    print(io, "role=$(Int(f.struct_role)), ")
-    print(io, "name=$(rpad(f.field_name, 30, " ")), ")
-    print(io, "type=$(rpad(f.type_name, 60, " "))")
-    print(io, "repetition=$(f.repetition),")
-    # print(io, "desc=$(f.field_desc),")
-end
+# function Base.show(io::IO, f::FieldRecord)
```

Comment:
Do you want to keep these comments? Just asking if it's an accidental commit ;)

Reply:
I do want to clean up these. they are useful identation for debugging but get in the way of displaying the larger RNTuple and RNTupleHeader